opentelemetry-sdk/src/opentelemetry/sdk/trace/export/__init__.py

```diff
@@ -76,3 +76,16 @@ def on_end(self, span: Span) -> None:
 
     def shutdown(self) -> None:
         self.span_exporter.shutdown()
+
+
+class ConsoleSpanExporter(SpanExporter):
+    """Implementation of :class:`Exporter` that prints spans to the console.
```

Comment:
```suggestion
    """Implementation of :class:`SpanExporter` that prints spans to the console.
```

Reply:
Thanks for the catch. I copied it the docstring from `InMemorySpanExporter`, now both have been fixed.